### PR TITLE
445 status message multiple lines of text

### PR DESCRIPTION
**Pull request by MikeDThorpe**
- implements Tom's refactoring suggestion to ensure text is aligned correctly on tablet < screen sizes

### CODE REVIEW

**Comment by tomyems** [comment]
Ah looks good thanks :) - I noticed that the spacing has changed on the other stories (which is a consequence of the changes I requested). Personally, I'm happy with making a breaking change, though we might need to amend the original stories from ` - due for review 31 May 2021` to `Due for review 31 May 2021`?

Probably one to show both to the design system working group

**Comment by MikeDThorpe** [comment]
@tomyems - where has spacing changed in the other stories? I noticed on smaller screens the Status Message with no 'extra content' and no 'actions' (a.k.a buttons) had uneven margin caused by the status text styling. So i moved the styling from the status text to the 'extra content' text so this would only be applied if the extra text was present. Perhaps i haven't applied the new styling correctly if it looks off - i so i can fix this if it's an issue.

**Comment by tomyems** [comment]
Looks good to me. It was about the spacing between the bold text and the '-'. Would be good to share in the design system working group to get their thoughts :)

**Review by tomyems — APPROVED** [review]
LGTM - does the version number need changing

**Comment by MikeDThorpe** [comment]
> LGTM - does the version number need changing

Where can i find the version number? And what should this be incremented too?

**Comment by tomyems** [comment]
> > LGTM - does the version number need changing
> 
> Where can i find the version number? And what should this be incremented too?

The status message has it's own version number https://github.com/UKHomeOffice/design-system/blob/master/components/status-message/package.json

However, I notice all the components have the same version number. I'm not sure how significant your changes are (and whether they're breaking changes, minor changes or bug fixes) to know how much to increment the version number by. I also don't know if it's okay to increase the version number of one component and not the rest of the design system.

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.3
+lockfileVersion: 5.4
```
Comment (daniel-ac-martin) [inline pnpm-lock.yaml:1]: We've not changed the dependencies so this file shouldn't need to change and should be removed from the PR.

File: components/status-message/assets/StatusMessage.scss [hunk 2]
```diff
@@ -5,67 +5,85 @@ $govuk-button-colour: govuk-colour("white");
 $govuk-button-text-colour: govuk-colour("black");
 
 .hods-status-message {
-  color: govuk-colour("white");
-  background-color: govuk-colour("blue");
-  @include govuk-responsive-padding(4, "all");
+	color: govuk-colour("white");
+	background-color: govuk-colour("blue");
+	@include govuk-responsive-padding(4, "all");
```
Comment (daniel-ac-martin) [inline components/status-message/assets/StatusMessage.scss:10]: There are lots of stylistic changes, such as this white space change. This should be removed from the PR as it makes it hard to review what has actually changed.

**Review by daniel-ac-martin — CHANGES_REQUESTED** [review]
The preview looks good but there are some issues in the code that should be addressed.

File: components/status-message/assets/StatusMessage.scss (hunk 2 above)
Comment (MikeDThorpe) [reply components/status-message/assets/StatusMessage.scss]: @daniel-ac-martin - this is from my local prettier settings which i can remove but not sure of the easiest way to revert them. Is there a formatting config file for the design system for contributors to use?

**Comment by MikeDThorpe** [comment]
@daniel-ac-martin - could you please advise on the correct way to handle the conflict in the pnpm-lock.yml file?

File: components/status-message/assets/StatusMessage.scss (hunk 2 above)
Comment (daniel-ac-martin) [reply components/status-message/assets/StatusMessage.scss]: >Is there a formatting config file for the design system for contributors to use?
Not currently. It's on my to-do list!

**Comment by daniel-ac-martin** [comment]
@MikeDThorpe: As we aren't changing any dependencies, the file shouldn't need to change. So I'd suggest that we just take the pnpm-lock.yaml from the master branch and commit that on top of this branch.

File: .jest-results.json [hunk 3]
```diff
@@ -1 +1 @@
...
...itles":["Header","when given valid props"],"duration":0,"failureMessages":[],"fullName":"Header when given valid props renders","location":null,"status":"passed","title":"renders"}],"endTime":1669287482582,"message":"","name":"/Users/michaelthorpe/Documents/project-work/hods-design-system/design-system/components/header/spec/Header.ts","startTime":1669287479924,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["DateTime","when given minimal valid props to display time then date"],"duration":2,"failureMessages":[],"fullName":"DateTime when given minimal valid props to display time then date renders","location":null,"status":"passed","title":"renders"},{"ancestorTitles":["DateTime","when given full valid props to display time then date"],"duration":1,"failureMessages":[],"fullName":"DateTime when given full valid props to display time then date renders","location":null,"status":"passed","title":"renders"}],"endTime":1669287482649,"message":"","name":"/Users/michaelthorpe/Documents/project-work/hods-design-system/design-system/components/date-time/spec/DateTime.ts","startTime":1669287480279,"status":"passed","summary":""}],"wasInterrupted":false,"coverageMap":{}}
```
Comment (daniel-ac-martin) [inline .jest-results.json:1]: Sorry can we remove this file from the PR. - It's a built file and we don't need to commit it.

(Sorry, I know I need to find a way to block this.)

File: components/status-message/src/StatusMessage.tsx [hunk 4]
```diff
@@ -14,23 +14,30 @@ export type StatusMessageProps = StandardProps & {
...
 
   return (
     <div {...attrs} className={classes()}>
-      <p>
-        {status}
-        {children && <span className={classes('extra')}> {children}</span>}
-      </p>
-      {actions && actions.length && <AnchorList items={actions} classBlock={classes('actions')} />}
+      <div>
+        <p className={classes('status')}>{status}</p>
```
Comment (daniel-ac-martin) [inline components/status-message/src/StatusMessage.tsx:35]: `{status}` isn't really a paragraph so I don't think we should use a `<p>` tag here.

I'd suggest we change this to a `<div>`.

File: components/status-message/assets/StatusMessage.scss [hunk 5]
```diff
@@ -9,17 +9,22 @@ $govuk-button-text-colour: govuk-colour("black");
   background-color: govuk-colour("blue");
   @include govuk-responsive-padding(4, "all");
 
-  p {
+  &__status {
+    margin: 0;
     font-weight: bold;
-  }
+  }
```
Comment (daniel-ac-martin) [inline components/status-message/assets/StatusMessage.scss:15]: Extraneous white-space.

**Review by daniel-ac-martin — CHANGES_REQUESTED** [review]
Sorry for the delay on reviewing this.

Just needs a few small changes, I think. Looks good otherwise.

**Review by tomyems — APPROVED** [review]

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**